.prettierrc

```diff
@@ -0,0 +1,18 @@
+{
+  “arrowParens”: “avoid”,
```

Comment:
this looks like it has a bunch of smart quotes.....  Can you convert it to JSON?

Also, in the PR, can you confirm that after you have this, running prettier on all files doesn't change anything?  If it does, then let's maybe re-PR updates on those (in a separate PR mid-stage)